providers/org/arxiv/harvester.py

```diff
@@ -1,7 +1,66 @@
-from share.harvest.harvester import OAIHarvester
+import logging
+import datetime
 
+from furl import furl
+from lxml import etree
 
-class ArxivHarvester(OAIHarvester):
-    rate_limit = (1, 3)
-    time_granularity = False
-    url = 'http://export.arxiv.org/oai2'
+from share import Harvester
+
+logger = logging.getLogger(__name__)
+
+
+class ArxivHarvester(Harvester):
+    namespaces = {
+        'ns0': 'http://www.w3.org/2005/Atom'
+    }
+    start_page_num = 0
+    url = 'http://export.arxiv.org/api/query?search_query=all&max_results=100&sortBy=lastUpdatedDate&sortOrder=descending'
```

Comment:
For niceness sake go ahead and just keep the url as `http://export.arxiv.org/api/query` and add the parameters using furl.

It looks like arxiv supports `https` but its not verified. Let's try to use SSL when available, even if its unverified its better than nothing.